components/nexus-bootstrap/src/main/java/org/sonatype/nexus/bootstrap/jetty/JettyServer.java

```diff
@@ -111,7 +127,7 @@ public Object run() {
     }
   }
 
-  private void doStart(boolean waitForServer) throws Exception {
+  private void doStart(boolean waitForServer, @Nullable final Runnable callback) throws Exception {
     if (thread != null) {
```

Comment:
Does this need to be made thread safe? (Can different threads try to start this thing?)